README.md

```diff
@@ -1,40 +1,42 @@
-# Engage Boilerplate
+# PlanIt
 
-## Installation
+Authors: Luke Melton, Kelly Dangora, Dan Leibensperger, Chris Bevevino
 
-For a copy of this repository please `Download ZIP` (found under the green `Code` button).
+# Description
 
-After unpacking the ZIP you can rename the folder/ directory for your new project.
+This app allows users to review planets.
 
-### Rename Boilerplate
+## Features
 
-The application will have all configurations set up with the name `planIt`. You can change the name to match and be consistent with the name of your project. To do this, use a global search (using the Search magnifying glass in VSCode) to look for all instances of `planIt` and replace it with the hyphenated name of your project (the search should find references to files with your database names, `package.json` files, etc.).
+- Users can view reviews of planets.
+- User can add a review for an existing planet.
+- User can edit their review for an existing planet.
+- User can add a new planet for review.
+- User can upvote and/or downvote reviews.
 
-## Adding to GitHub
+## Technologies Used
 
-1. Initialize your project with GitHub in the terminal:
+- Javascript
+- React.js
+- Express.js
+- Postgres
+- Objection.js
+- SQL
 
-   ```sh
-   git init
-   git add .
-   git commit -m "initial commit"
-   ```
+## Link to Deployed Site
+<>Link to site</>
```

Comment:
I know we mentioned this, but unless you plan on deploying it you can remove this